pkg/catalog/catalog.go

```diff
@@ -138,22 +137,18 @@ func getDefaultBuilderImages(client *occlient.Client) ([]CatalogImage, error) {
 	// Resultant imagestreams is list of imagestreams from current and openshift namespaces
 	imageStreams = append(imageStreams, openshiftNSImageStreams...)
 	imageStreams = append(imageStreams, currentNSImageStreams...)
-
 	var builderImages []CatalogImage
 
 	// Get builder images from the available imagestreams
 	for _, imageStream := range imageStreams {
 		var allTags []string
 		buildImage := false
-
 		for _, tag := range imageStream.Spec.Tags {
-
```

Comment:
@girishramnani you you need to revert these changes, you've deleted the newlines